docs/user_guide/index.rst

```diff
@@ -533,3 +533,66 @@ Then, follow the pattern of normal startup but specify your function in the :py:
 
 The benefit of this approach over manipulating the frame later is that the Seismogram can be considered frozen.
 Among other things, this means any Seismograph notebooks cells do not have a dependence on order and can be run multiple times.
+
+
+Create Custom Visualizations
```

Comment:
@firste - mostly looking for feedback on this blurb

Reply:
Looks good to me.